prestapyt/dict2xml.py

```diff
@@ -12,9 +12,14 @@
 
 from __future__ import unicode_literals
 from xml.dom.minidom import getDOMImplementation
-from past.builtins import basestring
 from builtins import str
 
+# past.builtins generates deprecated warning (import imp)
+try:
+    from __builtin__ import basestring
+except ImportError:
+    from past.types import basestring
```

Comment:
Why not `past.builtins`?

Reply:
Because past.builtins imports past.builtins.misc
![image](https://user-images.githubusercontent.com/10652233/184303238-04b9c5ad-5f28-410f-91ca-2a71ac47110f.png)
and past.builtins.misc imports from imp
![image](https://user-images.githubusercontent.com/10652233/184303410-50723bd0-9bdd-439d-9be4-229eb296b564.png)
and imp has deprecated warning
![image](https://user-images.githubusercontent.com/10652233/184303609-5b9978f7-1aee-475e-9cbd-0c66ed071da9.png)

Reply:
@pedrobaeza are you ok with the above explanation?